dashboards/mssql-server/alerts/documentation.md

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Can we rename this to `README.md` so GitHub will render it when viewing the folder?